MetricImperialConverter.java

```diff
@@ -0,0 +1,83 @@
+package com.myfitnesstracker.service;
+import java.util.Scanner;
+
+// class if needed to convert metric to imperial and vice versa, still needs to be tested fully for no errors
+
+public class MetricImperialConverter {
+
+    public void readInput() {
+        Scanner reader = new Scanner(System.in);
+
+        if (reader.hasNextLong()) {
+            long number = reader.nextLong();
+            if (reader.hasNext("\\b+(mil|in|inch|ft|foot|feet|yd|yard|ml|mile)\\b+")) {
+                String unit = reader.findInLine("\\b+(mil|in|inch|ft|foot|feet|yd|yard|ml|mile)\\b+");
+                double mm = toMm(number, unit);
+                System.out.println(number + " " + unit + " is:");
```

Comment:
Avoid System.out.println... no one will see this on a running Android app.  Consider logging instead.